sync.py

```diff
@@ -129,10 +129,16 @@ def _check_dir_for_changes(self, dir_path):
 
     def poll_for_changes(self, wait_time=1):
         while True:
-            self.pinger.ping()
-            self._check_dir_for_changes(self.base_path)
-            self._first_pass = False
-            time.sleep(wait_time)  # Poll for changes every `wait_time` seconds
+            try:
+                self.pinger.ping()
+                self._check_dir_for_changes(self.base_path)
+                self._first_pass = False
+            except requests.exceptions.ConnectionError:
+                print(f"Failed\nCould not reach {SERVER_URL}")
+            except Exception:
+                print("Unknown error")
+            else:
```

Comment:
BUT.... I think this might have caused our problem. I DDoS'ed our server when their internet connection went down 🤦 

Sorry!! I should have checked that you didn't have a session at the same time.